unit_tests/Common.d

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Feb 14, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+
+import std.stdio;
+import std.random;
+import std.file;
+import std.path;
+import std.conv;
+import std.algorithm;
+import std.array;
+import clang.c.Index;
+
+import dstep.translator.Translator;
+import dstep.translator.Output;
+
+public import clang.Compiler;
+public import clang.Cursor;
+public import clang.Index;
+public import clang.TranslationUnit;
+public import clang.Token;
+
+Index index;
+
+static this() 
+{
+    index = Index(false, false);
+}
+
+TranslationUnit makeTranslationUnit(string C) 
+{
+    return TranslationUnit.parseString(index, C, [], null, CXTranslationUnit_Flags.CXTranslationUnit_DetailedPreprocessingRecord);
+}
+
+string translate(TranslationUnit translationUnit) 
+{
+    resetOutput();
+    auto translator = new Translator(translationUnit);
+    return translator.translateToString();
+}
+
+class TranslateAssertError : object.Error 
+{
+    this (string message, string file, ulong line) 
+    {
+        super(message, file, line);
+    }
+}
+
+void assert_translates(string C, string D, string file = __FILE__, size_t line = __LINE__) 
```

Comment:
* Please use camel case for the method name
* Lower case identifiers for the parameters